src/test/java/de/tracetronic/jenkins/plugins/ecutest/tool/installation/ETInstallationTest.java

```diff
@@ -28,16 +29,21 @@ public void testInstallation() {
         assertEquals("ECU-TEST", inst.getName());
         assertEquals("C:\\ECU-TEST", inst.getHome());
         assertEquals("ECU-TEST.Application", inst.getProgId());
+        assertEquals(0, inst.getTimeout());
+        assertFalse(inst.isRegisterComServer());
     }
 
     @Test
     public void testInstallationWithCustomProgId() {
         final ETInstallation inst = new ETInstallation("ECU-TEST", "C:\\ECU-TEST",
-            Collections.singletonList(new ETToolProperty("ECU-TEST6.Application", 120)));
+            Collections.singletonList(new ETToolProperty("ECU-TEST6.Application", 120, true)));
```

Comment:
mixed test case - this unit test tests custom `progId`, the test of `isRegisterComServer` was added additionally. Separate test case for isRegisterComServer?